src/rgw/rgw_quota.cc

```diff
@@ -944,10 +951,12 @@ class RGWQuotaHandlerImpl : public RGWQuotaHandler {
      * fetch that info and not rely on cached data
      */
 
+    const char *prefix = "rgw quota handler: ";
+    const DoutPrefix dp(store->ctx(), dout_subsys, prefix);
```

Comment:
@cbodley `store` isn't used at all in `check_quota`. Are we sure that it will be initialized whenever `check_quota()` is run?